packages/cli/src/schemas.ts

```diff
@@ -28,7 +28,7 @@ const CONTROLLER_IMAGE_DEFAULT = `opstrace/controller:${BUILD_INFO.VERSION_STRIN
 // `us-west-2a` encodes availability zone "a" within that region.
 export const infraConfigSchemaAWS = yup
   .object({
-    instance_type: yup.string().default("t3.2xlarge"),
+    instance_type: yup.string().default("t3.xlarge"),
```

Comment:
If we make this change a documentation upadte has to be part of this :)

https://github.com/opstrace/opstrace/blob/3f405cd4baa709c5d624d8966b8e2820b28ea37f/docs/references/cluster-configuration.md#aws

Reply:
edit: oh, the default values are not yet explicitly documented. OK.